pkg/operator/ceph/cluster/mgr/config.go

```diff
@@ -55,16 +56,15 @@ func (c *Cluster) dashboardPort() int {
 	return c.dashboard.Port
 }
 
-func (c *Cluster) generateKeyring(m *mgrConfig) error {
+func (c *Cluster) generateKeyring(m *mgrConfig) (string, error) {
 	user := fmt.Sprintf("mgr.%s", m.DaemonID)
 	/* TODO: the access string here does not match the access from the keyring template. should they match? */
```

Comment:
If you feel like you have time, while you are here editing this function, could you remove this TODO as well since this is no longer true?